AirCasting/MicrophoneSession/MicrophoneManager.swift

```diff
@@ -110,7 +110,7 @@ private extension MicrophoneManager {
 
     func createMeasurementStream(for session: Session) throws -> MeasurementStreamLocalID {
         let stream = MeasurementStream(id: nil,
-                                       sensorName: "Phone Microphone",
+                                       sensorName: Strings.SensorsData.microphone,
```

Comment:
I'm not sure but shouldn't `Strings` contain only text we display for users? 
Maybe we could make a constant for this sensor name?

Reply:
It needs to be accessible from everywhere, maybe let's just name String struct well, what do You think?

Reply:
I agree with @smialko . We need to introduce a completely new namespace for this. Something like:
`Constants.SensorNames.microphone` or something